cypress/integration/dashboard_spec.ts

```diff
@@ -53,7 +53,7 @@ describe('The Dashboard', () => {
     // the text "Loading..." (displays a spinner)
     cy.waitForAllGetRequests()
 
-    cy.getBySel('token-row-sell-max-button').click()
+    cy.getBySel('token-row-sell-max-button').should('be.disabled')
```

Comment:
I'm not sure this should be disabled - we have a FOX balance mocked.
I suspect this is why CI is failing.

Reply:
it was originally failing as a click with my changes. Will play around with it more

Reply:
Have a play Cypress in headed mode - we've mocked a value of `1,450` FOX in the wallet, and so "Max" should, and appears to, work correctly when playing with the mocked state interactively.

Interactive mode can be used once the tests have finished running - you're left in the final state.

<img width="337" alt="Screen Shot 2022-05-19 at 8 49 55 am" src="https://user-images.githubusercontent.com/97164662/169167976-bd740b35-4225-462c-9869-ec699d6b7ef7.png">

Reply:
🪄